### PR TITLE
Minor cleanup in async branch in preparation for merge to develop

### PR DESCRIPTION
**Pull request by derobins**

### CODE REVIEW

**Review by qkoziol — APPROVED** [review]
These all look good, but I've changed the H5ES* routines quite a bit, so if you could wait until after the review at 3pm today before merging them in, that would be very helpful.

**Comment by derobins** [comment]
> These all look good, but I've changed the H5ES* routines quite a bit, so if you could wait until after the review at 3pm today before merging them in, that would be very helpful.

Let me know when you are ready and I'll pull whatever changes you've made into my branch and tidy up. Will you want to click the merge button or should I?

**Pull request merged**
